test/testModel.cc

```diff
@@ -776,32 +776,15 @@ SUITE(Canvas)
         godley->table.cell(2,2)="y";
         godley->update();
 
-        
+        // Unit test greatly simplified since flow/stock vars no longer copied via temporary group. For ticket 1039
```

Comment:
This comment is not really helpful for down the track. Add it to the ticket discussion if you want to comment on it.

Reply:
I've removed it, thanks.